content/conduct/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Code of conduct"
+date: 2018-11-14T16:28:00-07:00
+draft: false
+weight: 150
+---
+
+- [Our pledge](#our-pledge)
+- [Our Standards and Values](#our-standards-and-values)
+- [Our Responsibilities](#our-responsibilities)
+- [Scope](#scope)
+- [Enforcement](#enforcement)
+- [Attribution](#attribution)
+
+#### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers of OpenCensus pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation. People violating this code of conduct may be banned from the community and all its events.
+
+#### Our Standards and Values
+
```

Comment:
We should add one additional line at the top that @adriancole proposed and I expanded upon:

- Be honest: be open and transparent in your dealings with other community members. Be clear about your goals, motivations, and thoughts.

Reply:
This should be added as the top bullet under _Our Standards and Values_

Reply:
Oh, I was copying it verbatim from the previously linked source. If you'd like to update your PR #453 with the same structure as this one i.e. `/conduct` instead of `/community/codeofconduct` we can then instead move to yours as I don't know exactly where to place that line :)